projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -299,6 +311,67 @@ public function handle_request_revoke_invite() {
 			wp_die();
 		}
 
+		/**
+		 * Handles resend user invite.
+		 */
+		public function handle_request_resend_invite() {
+			check_admin_referer( 'jetpack-sso-resend-user-invite', 'resend_invite_nonce' );
+			$nonce = wp_create_nonce( 'jetpack-sso-invite-user' );
+			if ( ! current_user_can( 'create_users' ) ) {
+				$query_params = array(
+					'jetpack-sso-invite-user'  => 'failed',
+					'jetpack-sso-invite-error' => 'invalid-user-permissions',
+					'_wpnonce'                 => $nonce,
+				);
+				return self::create_error_notice_and_redirect( $query_params );
+			} elseif ( isset( $_GET['invite_id'] ) ) {
+				$invite_slug = sanitize_text_field( wp_unslash( $_GET['invite_id'] ) );
+				$blog_id     = Jetpack_Options::get_option( 'id' );
+				$url         = '/sites/' . $blog_id . '/invites/resend';
+				$response    = Client::wpcom_json_api_request_as_user(
+					$url,
+					'v2',
+					array(
+						'method' => 'POST',
+					),
+					array(
+						'invite_slug' => $invite_slug,
+					),
+					'wpcom'
+				);
+				$body        = json_decode( $response['body'] );
+				$status_code = json_decode( $response['response']['code'] );
+
+				if ( 200 !== $status_code ) {
+					$message_type = $status_code === 404 ? 'invalid-invite' : ''; // empty is the general error message
+					$query_params = array(
+						'jetpack-sso-invite-user'  => 'failed',
+						'jetpack-sso-invite-error' => $message_type,
+						'_wpnonce'                 => $nonce,
+					);
+					return self::create_error_notice_and_redirect( $query_params );
+				}
+				$query_params = array(
+					'jetpack-sso-invite-user' => $body->success ? 'reinvited-success' : 'failed',
+					'_wpnonce'                => $nonce,
+				);
+
+				if ( ! $body->success ) {
+					$query_params = array(
```

Comment:
Lets update this to add `'jetpack-sso-invite-error' => $body->errors[0]` to the query_params array instead of overriding it.